rastervision/data/raster_source/image_source_config.py

```diff
@@ -82,6 +82,7 @@ def from_proto(self, msg):
             .with_uri(msg.image_file.uri)
 
     def with_uri(self, uri):
+        """Set URI for a GeoTIFF used to read labels."""
```

Comment:
ImageSource is for non-georeferenced image files including tifs and pngs. not sure if rasterio can open jpgs.

Reply:
Also, this is used to read raster data, not necessarily labels.

Reply:
Done.